restcomm/restcomm.testsuite/src/test/java/org/restcomm/connect/testsuite/http/CallsEndpointTest.java

```diff
@@ -82,11 +84,52 @@ public void getCallsList() {
         assertTrue(totalSize == 448);
     }
 
+    @Test
+    public void getCallsListUsingSorting() {
```

Comment:
is confirmed locally about test results https://cxs.restcomm.com/job/Restcom-CI-Multibranch/job/PR-2977/1/testReport/

could you please check those?

Reply:
My bad, I was planning to check the results but totally forgot, will check them

Reply:
It's weird though because I ran the CallsEndpointTest.java multiple times with my changes and it was working fine. Anyway, let me get back to you on that

Reply:
beware, soemtimes tests results depends on the current DB state, so running test cases alone, and runnig the full test class, may lead to different resutls.

Please ensure running the whole test class is providing stable results.

Also please notice the usage of  @FixMethodOrder(MethodSorters.NAME_ASCENDING) annotation in test class, to enforce a predictable order of test cases execution,and hence predictable results. Again reusing DB state from test case to test case is bad practice, but this is what it is now...

Reply:
Sure, will keep in mind